thirdparty/itt_collector/CMakeLists.txt

```diff
@@ -14,6 +14,4 @@
 # limitations under the License.
 # ******************************************************************************
 
-if(ENABLE_PROFILING_ITT AND SELECTIVE_BUILD STREQUAL "COLLECT")
-    add_subdirectory(sea_itt_lib)
-endif()
+add_subdirectory(sea_itt_lib)
```

Comment:
It can be built only if `ENABLE_PROFILING_ITT ` is enabled.

BTW, do we sure that OV package won't include this library if we always build it?

Reply:
May be is better to add unit test for sea_itt_lib validation?